src/server/game/Spells/SpellMgr.cpp

```diff
@@ -6295,7 +6298,8 @@ void SpellMgr::LoadDbcDataCorrections()
     properties->Type = SUMMON_TYPE_TOTEM;
     properties = const_cast<SummonPropertiesEntry*>(sSummonPropertiesStore.LookupEntry(647)); // 52893
     properties->Type = SUMMON_TYPE_TOTEM;
-
+    if (properties = const_cast<SummonPropertiesEntry*>(sSummonPropertiesStore.LookupEntry(628))) // Hungry Plaguehound
```

Comment:
This is causing an error. To compare properties with the ones you're looking up too, you need '==' instead of '=' else you are attributing a new value to properties instead. I'm sure you didn't need the explanation but just a reminder because you probably forgot or miss-typed ;)

Reply:
@FrancescoBorzi He actually means it by assigning the value to properties to check if it's true, he's just missing parentheses you can close this